test/into.js

```diff
@@ -0,0 +1,50 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('into', function() {
+    var add = R.curry(function(a, b) {return a + b;});
```

Comment:
Not particularly important to worry about here, but is there any reason not to use `R.add` here?